src/components/breadcrumbs/_variables.scss

```diff
@@ -12,8 +12,17 @@
 $ouiBreadcrumbSpacing: $ouiSizeS !default;
 $ouiBreadcrumbTruncateWidth: $ouiSize * 10 !default;
 
+$ouiBreadcrumbBlueBackground: #B9D9EB !default;
+$ouiBreadcrumbGrayBackground: #D9E1E2 !default;
+$ouiBreadcrumbCollapsedLink: #002A3A !default;
+$ouiBreadCrumbHoverColor: #535861 !default;
```

Comment:
If the defaults are set here, do we need to also hardcode the values into the theme?

Reply:
I believe so, but this is more of an issue with the old theming system, which we are working towards having a cleaner, more pleasant solution for.

Reply:
Is this related @BSFishy: https://github.com/opensearch-project/oui/issues/212?

If so, left a comment there.